com.hoptodesk.HopToDesk.json

```diff
@@ -0,0 +1,264 @@
+          ],
+          "sources" : [
+            {
+              "url" : "https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.112.tar.gz",
+              "sha256" : "7d550dccdfcd286e33895501829ed971eeb65c614e73aadb4a08aeef719b143a",
+              "type" : "archive"
+            }
+          ],
+          "cleanup" : [
+            "*.la",
+            "/bin",
+            "/etc",
+            "/include",
+            "/libexec",
+            "/share/gtk-doc",
+            "/share/man"
+          ]
+        },
+        {
+          "name" : "libdbusmenu",
+          "config-opts" : [
+            "--disable-static",
+            "--disable-gtk-doc",
+            "--enable-introspection=no",
+            "--disable-vala",
+            "--disable-dumper",
+            "--disable-tests",
+            "--with-gtk=3"
+          ],
+          "sources" : [
+            {
+              "url" : "https://launchpad.net/libdbusmenu/16.04/16.04.0/+download/libdbusmenu-16.04.0.tar.gz",
+              "sha256" : "b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a",
+              "type" : "archive"
+            }
+          ],
+          "build-options" : {
+            "cflags" : "-Wno-error",
+            "env" : {
+              "HAVE_VALGRIND_FALSE" : "#",
+              "HAVE_VALGRIND_TRUE" : ""
+            }
+          },
+          "cleanup" : [
+            "/include",
+            "/libexec",
+            "/lib/pkgconfig",
+            "/lib/*.la",
+            "/share/doc",
+            "/share/libdbusmenu",
+            "/share/gtk-doc",
+            "/share/gir-1.0"
+          ]
+        },
+        {
+          "name" : "libindicator",
+          "config-opts" : [
+            "--disable-static",
+            "--disable-tests",
+            "--with-gtk=3"
+          ],
+          "sources" : [
+            {
+              "url" : "https://launchpad.net/libindicator/12.10/12.10.1/+download/libindicator-12.10.1.tar.gz",
+              "sha256" : "b2d2e44c10313d5c9cd60db455d520f80b36dc39562df079a3f29495e8f9447f",
+              "type" : "archive"
+            },
+            {
+              "commands" : [
+                "# FTBFS fix",
+                "sed -e '/LIBINDICATOR_LIBS/ s/\\$LIBM/ $LIBM/' -i configure.ac"
+              ],
+              "type" : "shell"
+            },
+            {
+              "commands" : [
+                "autoreconf -sfi"
+              ],
+              "dest-filename" : "autogen.sh",
+              "type" : "script"
+            }
+          ],
+          "build-options" : {
+            "cflags" : "-Wno-error"
+          },
+          "cleanup" : [
+            "/include",
+            "/libexec",
+            "/lib/pkgconfig",
+            "/lib/*.la",
+            "/share/libindicator",
+            "/share/gtk-doc",
+            "/share/gir-1.0"
+          ]
+        }
+      ]
+    },
+    {
+      "name" : "xdotool",
+      "buildsystem" : "simple",
```

Comment:
This should not be simple use `no-autogen`, and `make-args`

Reply:
Updated